charts/simpledataexchanger/values.yaml

```diff
@@ -0,0 +1,259 @@
+#################################################################################
+# Copyright (c) 2023 T-Systems International GmbH
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+################################################################################
+# backend for SDE
+replicaCount: 1
+
+sdepostgresql:
+# -- PostgreSQL chart configuration
+  # Switch to enable or disable the PostgreSQL helm chart
+  enabled: true
+  fullnameOverride: "product-sde-postgres"
+  auth:
+    username: "sdeuser"
+    port: 5432
+    database: "sdedb"
+    existingSecret: "sde-postgresql-secrets"
+    secretKeys:
+      adminPasswordKey: "postgres-password"
+      userPasswordKey: "password"
+
+backend:
+  image:
+    repository: ghcr.io/catenax-ng/tx-managed-simple-data-exchanger-backend
+    pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+    tag: ""
+
+  imagePullSecrets: []
+
+  nameOverride: ""
+
+  fullnameOverride: ""
+
+  portContainer: 8080
+
+  podAnnotations: {}
+
+  podSecurityContext:
+    fsGroup: 2000
+
+  securityContext:
+    allowPrivilegeEscalation: false
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
```

Comment:
These should be not commented by default as they are required by the [TRGs](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-03/#implementation).

```
  runAsNonRoot: true
  runAsUser: 1000
```

Reply:
Also make sure the user id is the same as the user set in the docker file.

Reply:
Sure. Checking

Reply:
changed as requested.